src/homework-1/task2/task-2.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+const csv = require('csvtojson')
+
+const readStream = fs.createReadStream('./resources/books.csv')
+const writeStream = fs.createWriteStream('./resources/books.txt')
+
+csv()
+  .fromStream(readStream)
+  .on('data', data => writeStream.write(data, 'utf-8'))
+  .on('done', error => processFinishedHandler(error))
+  .then()
```

Comment:
Agree. I've added `then` because webstorm highlight unused promise. 
![image](https://user-images.githubusercontent.com/37336437/143273199-0fc1fd5f-3d6e-4034-8eb4-b7e9c1687d79.png)